R/autoplot.summary.seroincidence.by.R

```diff
@@ -41,22 +42,31 @@
 #'
 #' est2sum <- summary(est2)
 #'
-#' est2sum |> strat_ests_scatterplot(
+#' est2sum |> autoplot(
 #'     type ="scatter",
 #'     xvar = "ageCat",
 #'     color_var = "catchment",
 #'     CIs = TRUE,
 #'     group_var = "catchment")
 #'
+#' est2sum |> autoplot(
+#'     type ="bar",
+#'     yvar = "ageCat",
+#'     color_var = "catchment",
+#'     CIs = TRUE)
+#'
 autoplot.summary.seroincidence.by <- function(
     object,
-    type = "scatter",
-    ...) {
+    type,
+    ...
+) {
```

Comment:
```suggestion
    ...) {
```
This change should eliminate the indentation lint (I think this is [an error in the `lintr` package](https://github.com/r-lib/lintr/issues/2830), but let's listen to it for now)